src/coreclr/jit/instr.h

```diff
@@ -327,6 +313,15 @@ enum insOpts : unsigned
 #endif
 };
 
+enum insGroupOpts : unsigned
```

Comment:
I would rename this to `insScalableOpts` to call out that this is specific to SVE